### PR TITLE
Updated the URL of my guide

### PR DESCRIPTION
**Pull request by aborruso**
We have changed the URL and all the style of the guide

- [ ] If contributing a core loader, [the loader checklist](https://visidata.org/docs/contributing#loader) was referenced.
- [ ] If registering an external plugin, [the plugin checklist](https://visidata.org/docs/contributing#plugins) was referenced.

### CODE REVIEW

**Review by saulpw — APPROVED** [review]

**Comment by aborruso** [comment]
Thank you @saulpw 

I do not see it in https://www.visidata.org/docs/

The update is not automatic after the PR approval (some github action that starts)?

**Comment by anjakefala** [comment]
@aborruso The website documentation will only update after the VisiData release! The website is built at https://github.com/visidata/visidata.org. =)

**Comment by aborruso** [comment]
> The website documentation will only update after the VisiData release! The website is built at https://github.com/visidata/visidata.org. =)

@anjakefala thank you.
I don't know if I understand you: should  I wait for the next release? Or can I change something in this other repo?

**Comment by anjakefala** [comment]
Ah yes! I can also update the website right now, since your change is on the `stable` branch. There is nothing needed for you to do. =)

**Comment by anjakefala** [comment]
It is now updated: https://www.visidata.org/docs/

**Pull request merged**
